src/Lykke.Logs/Obsolete/Slack/LykkeLogToSlack.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using System;
 using System.Text;
 using System.Threading.Tasks;
+using AsyncFriendlyStackTrace;
 using JetBrains.Annotations;
 using Common.Log;
+using Lykke.Common;
+using Lykke.Common.Log;
 using Lykke.SlackNotifications;
+using Microsoft.Extensions.Logging;
 
 namespace Lykke.Logs.Slack
 {
     /// <summary>
     /// Logs entries to the specified Slack channel. Which types of entries should be logged, can be configured
     /// </summary>
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
fixed